flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/checkpointing/AlignedController.java

```diff
@@ -39,6 +40,8 @@
 /** Controller for aligned checkpoints. */
 @Internal
 public class AlignedController implements CheckpointBarrierBehaviourController {
+    private final boolean failOnUnalignedBarriers;
```

Comment:
This will need to be reimplemented on top of @dawidwys changes :(

Reply:
I'd expect some changes to the input side anyways.

Reply:
Changed it by using a forced aligned checkpoint. So not valid anymore.